model/project_ref.go

```diff
@@ -849,6 +849,9 @@ func (p *ProjectRef) createNewRepoRef(u *user.DBUser) (repoRef *RepoRef, err err
 	if !utility.StringSliceContains(repoRef.Admins, u.Username()) {
 		repoRef.Admins = append(repoRef.Admins, u.Username())
 	}
+	if repoRef.Id != "" {
```

Comment:
This isn't really necessary, since we create the repoRef ourselves in this function